test/index.js

```diff
@@ -26,6 +26,32 @@ lab.experiment('Rendr proxy adapter', function () {
 
         var output = adapter.apiDefaults(api, req);
 
+        Code.expect(output.headers).to.not.include('Cookie');
+
+        return done();
+    });
+
+    lab.test('keeps all request cookies with cookie-parser', function (done) {
+
+        var adapter = new ProxyAdapter({
+            default: {
```

Comment:
Any reason for the change in the interface? Adding the `default` thingy? Because it's a breaking change.